gohermes/mid_hermes.go

```diff
@@ -92,14 +79,17 @@ func (mh *MidHermes) AddBlock(id []byte, data []byte, meta []byte) error {
 }
 
 func (mh *MidHermes) ReadBlock(id []byte) ([]byte, []byte, error) {
+	if len(id) == 0 {
+		return nil, nil, ErrEmptyParameter
+	}
 	var block, meta *C.uint8_t
 	var block_length, meta_length C.size_t
-	if 0 != C.mid_hermes_read_block(
+	if status := C.mid_hermes_read_block(
```

Comment:
is it okay in Go to mix assignment and if statement? as for me, it's easier to read them separately. it's just imho, not critical

Reply:
it's ok when this variable doesn't used anywhere more and accessible only in if scope... but yeah, maybe it's better to split...

Reply:
maybe during next refactoring :)